internal/app/mistapiconnector/mistapiconnector_app.go

```diff
@@ -473,6 +473,77 @@ func (mc *mac) generateRouteKeys(stream *livepeer.CreateStreamResp) []string {
 	return keys
 }
 
+func (mc *mac) triggerUserNew(w http.ResponseWriter, r *http.Request, lines []string, rawRequest string) bool {
+	if len(lines) != 6 {
+		glog.Errorf("Expected 6 lines, got %d, request \n%s", len(lines), rawRequest)
+		w.WriteHeader(http.StatusBadRequest)
+		w.Write([]byte("false"))
+		return false
+	}
+
+	playbackId := lines[0]
+	stream, err := mc.lapi.GetStreamByPlaybackID(playbackId)
+	if err != nil || stream == nil {
+		glog.Errorf("Error getting stream info from Livepeer API playbackId=%s err=%v", playbackId, err)
+		w.WriteHeader(http.StatusNotFound)
+		w.Write([]byte("false"))
+		return false
+	}
+
+	userId := stream.UserID
+	userWebhooks, err := mc.lapi.GetWebhooksByUserId(userId, "user.new")
+	if err != nil {
+		glog.Errorf("Error getting webhooks for user %s err=%v", userId, err)
+		w.WriteHeader(http.StatusInternalServerError)
+		w.Write([]byte("false"))
+		return false
+	}
+
+	if len(userWebhooks) == 0 {
+		glog.V(model.DEBUG).Infof("No user.new webhooks for user %s", userId)
+		w.WriteHeader(http.StatusOK)
+		w.Write([]byte("false"))
+		return false
+	}
+
+	for _, userWebhook := range userWebhooks {
```

Comment:
We should probably call these in parallel (or not, if we only expect users to have 1 webhook anyway). It's easy to do it in Go though with an `errgroup`, [check it out](https://pkg.go.dev/golang.org/x/sync/errgroup).